config/jobs/cert-manager/approver-policy/cert-manager-approver-policy-presubmits.yaml

```diff
@@ -6,12 +6,15 @@ presubmits:
     always_run: true
     max_concurrency: 8
     annotations:
-      testgrid-create-test-group: 'false'
+      preset-go-cache: "true"
+      preset-local-cache: "true"
     spec:
       containers:
-      - image: eu.gcr.io/jetstack-build-infra-images/golang-dind:20231115-6d9432b-1.21.3
+      - image: eu.gcr.io/jetstack-build-infra-images/make-dind:20231115-8c0910c-bookworm
         args:
+        - runner
         - make
+        - vendor-go
```

Comment:
vendor go instead of using golang-dind

Reply:
This won't work if `make` is run with parallelism enabled:
 * https://www.gnu.org/software/make/manual/make.html#Parallel

It'll start on the `verify` target before finishing the `vendor-go` target.

And I assume that you will want to use parallelism to speed up the builds in future.